libraries/hybrids/src/basic-tests.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from "chai";
+import { define } from "hybrids";
+
+import {
+  ComponentWithoutChildren,
+  ComponentWithChildren,
+  ComponentWithChildrenCount,
+  ComponentWithDifferentViews,
+  ComponentWithProperties,
+  ComponentWithDeclarativeEvent,
+} from "./components";
+
+describe("basic support", function() {
+  // Setup the test harness. This will get cleaned out with every test.
+  const app = document.createElement("div");
+  app.id = "app";
+  document.body.appendChild(app);
+
+  let root; // This will hold the actual element under test.
+
+  beforeEach(function() {
+    root = document.createElement("div");
+    app.appendChild(root);
+  });
+
+  afterEach(function() {
+    app.innerHTML = "";
+    root = null;
+  });
+
+  describe("no children", function() {
+    define("component-without-children", ComponentWithoutChildren);
+
+    it("can display a Custom Element with no children", function(done) {
+      this.weight = 3;
+      root.appendChild(document.createElement("component-without-children"));
+
+      requestAnimationFrame(() => {
+        let wc = root.children[0].shadowRoot.querySelector("#wc");
```

Comment:
I will change that, but it is actually equal because `children` returns only elements (https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children).